docs/updates/2019-02.md

```diff
@@ -0,0 +1,93 @@
+# February 2019
+
+Welcome to the February 2019 Qlik Core update. We're highlighting a couple of important changes and
+improvements which we hope you will like.
+
+## Usability and design changes in catwalk
+
+Since being moved to stable last month, we've made several small usability improvements and design changes along with a
+couple of bug fixes.
+
+### QlikView select excluded coloring
```

Comment:
I don't like the "QlikView"-thingie. Use "Showing selected but excluded" or sth instead.

Reply:
Agreed, that's obvious in the paragraphs below anyway

Reply:
Agree!